Fixed syntax errors while minifying jquery-3.2.1

awegmueller opened the pull request:
Problem was that jQuery 3.2.1 uses a property named 'throws', which is a
future reserved keyword in YUI compressor. With the patch the check for
the throws keyword has been removed, so the property is treated like any
other regular property.

awegmueller committed:
Fixed syntax errors while minifying jquery-3.2.1

Problem was that jQquery uses a property named 'throws', which is a
future reserved keyword in YUI compressor. With the patch the check for
the throws keyword has been removed, so the property is treated like any
other regular property.

Set version to 2.4.9-BSI-1.

OlivierJaquemet commented:
Thanks a lot for providing a fix for this issue.
Too bad yuicompressor is no longer maintained and no one merge PR :cry:

padyx commented:
@OlivierJaquemet A built version of this pull request is deployed at Bintray.
If this fix is the only one you need, you might be happy with it.

Please note: That version is **not** an official build of yuicompressor.
There is no further development or support for that version, and it might go away at any time.

https://bintray.com/bsi-business-systems-integration-ag/scout-eclipse-org/yuicompressor

OlivierJaquemet commented:
Thank you @padyx, however we have already forked and integrated your fix in a custom build (with other modification related to build process, to prevent conflit with newer version of Rhino).
https://github.com/jalios/yuicompressor

cguglielmo committed:
Remove final and native from the list of reserved keywords

jquery-3.4.1 uses final as variable name -> minifying with yui fails.
The same happens if a variable called native would be used. Both
words are no official keywords anymore and can officially be used
as variable names -> support it.

Updated version to 2.4.9-BSI-2.
